### ISSUE TITLE
[Bug]  Missing libraries on Windows install

### ISSUE THREAD

**Issue by chaedri** [OSGeo/grass#2013]
**Describe the bug**
Installing grass-dev through the Windows Standalone installer results in missing libraries: libgomp, libfortran, libquadmath, and proj

**To Reproduce**

1. Install grass-dev through the Windows Standalone installer (https://grass.osgeo.org/download/windows/#standalone-installers)
2. Open nc_spm_08_grass7 example location
2. Run `r.surf.random out="uniform" min=-2 max=2`
3. See error: missing libgomp and libfortran
4. After finding and installing those libraries, run `r.surf.random out="uniform" min=-2 max=2` again
5. See error: missing libquadmath
6. To see missing proj library, run `m.proj -o coordinates=638300,220750`

**System description (please complete the following information):**

- Operating System: Windows 10 Home
- GRASS GIS version: GRASS GIS 8.0.dev (7be7cf9b0)

**Comment by chaedri** [comment]
This issue is still occurring for me, even with the latest GRASS-dev install. Has anyone else been able to reproduce it? Or tested it and was unable to reproduce it? Perhaps it is just an issue on my end.

**Comment by hellik** [comment]
Are the MS runtimes,  provided by the standalone installer, installed ?

**Comment by wenzeslaus** [comment]
> Are the MS runtimes, provided by the standalone installer, installed ?

Are you asking about checking that checkbox or about checking presence of some DLLs on the system? ...I just saw someone with the `libgfortran=5.dll was not found` error message dialog.

**Comment by hellik** [comment]
>Are you asking about checking that checkbox

yes, checking the check box

**Comment by hellik** [comment]
>Installing grass-dev through the Windows Standalone installer results in missing libraries: libgomp, libfortran, libquadmath,

these libraries are in MSYS2.

https://wingrass.fsv.cvut.cz/grass81/

these winGRASS8.1dev standalones works here for me. 

now testing https://grass.osgeo.org/grass80/binary/mswindows/native/WinGRASS-8.0.0RC2-1-Setup.exe

**Comment by hellik** [comment]
```
System Info                                                                     
GRASS Version: 8.0.0RC2                                                         
Code revision: fe5d1c795                                                        
Build date: 2022-01-17                                                          
Build platform: x86_64-w64-mingw32                                              
GDAL: 3.4.1                                                                     
PROJ: 8.2.1                                                                     
GEOS: 3.10.0                                                                    
SQLite: 3.35.2                                                                  
Python: 3.9.5                                                                   
wxPython: 4.1.1                                                                 
Platform: Windows-10-10.0.19044-SP0   
```

```
r.surf.random out=uniform min=-2 max=2                                          
r.surf.random komplett. Rasterkarte <uniform> erzeugt.
```

```
m.proj -o coordinates=638300,220750                                             
ERROR: cs2cs program not found, install PROJ first: 
```

does PROJ: 8.2.1  still provide cs2cs?

**Comment by hellik** [comment]
> does PROJ: 8.2.1 still provide cs2cs?

yes, it does. testing on the OSGeo4W v2 shell
```
C:\OSGeo4W>cs2cs
Rel. 8.2.1, January 1st, 2022
usage: cs2cs [-dDeEfIlrstvwW [args]]
              [[--area name_or_code] | [--bbox west_long,south_lat,east_long,north_lat]]
              [--authority {name}] [--accuracy {accuracy}] [--no-ballpark]
              [+opt[=arg] ...] [+to +opt[=arg] ...] [file ...]
```

so it's missing in the standalone . could you open a separate issue?

**Comment by hellik** [comment]
now testing with

```
System Info                                                                     
GRASS Version: 8.1.dev                                                          
Code revision: d21dd0a03                                                        
Build date: 2022-01-19                                                          
Build platform: x86_64-w64-mingw32                                              
GDAL: 3.4.1                                                                     
PROJ: 8.2.1                                                                     
GEOS: 3.10.2                                                                    
SQLite: 3.35.2                                                                  
Python: 3.9.5                                                                   
wxPython: 4.1.1                                                                 
Platform: Windows-10-10.0.19044-SP0 
```
```
r.surf.random out=uniform2 min=-2 max=2                                         
r.surf.random komplett. Rasterkarte <uniform2> erzeugt.
```

```
m.proj -o coordinates=638300,220750                                             
ERROR: cs2cs program not found, install PROJ first: 
```

```
C:\Users\xyz>cs2cs
Der Befehl "cs2cs" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
```

same here, it seems not all _PROJ_ tools are correctly packaged in the standalone installer. 

@landam: could you check your packaging environment?

**Comment by hellik** [comment]
>Installing grass-dev through the Windows Standalone installer results in missing libraries: libgomp,

libgomp is the GNU implementation of OpenMP. 

we're shipping libomp.dll provided by mingw-w64-openmp, it's the LLVM/OpenMP implementation. 

not sure why libgomp is here.... maybe (sub)dependency issue ,,,,

**Comment by hellik** [comment]
>m.proj -o coordinates=638300,220750                                             
>ERROR: cs2cs program not found, install PROJ first: 

we're shipping only _PROJ_ library in standalone, but not the _PROJ_  tools e.g. proj, cs2cs, geod

**Comment by hellik** [comment]
> > m.proj -o coordinates=638300,220750
> > ERROR: cs2cs program not found, install PROJ first:
> 
> we're shipping only _PROJ_ library in standalone, but not the _PROJ_ tools e.g. proj, cs2cs, geod

an issue or PR would be good.

**Comment by chaedri** [comment]
> Are the MS runtimes, provided by the standalone installer, installed ?

This was partly my issue - I installed with the link to 8.1dev above, ensured the Runtime option was checked and r.surf.random worked. Thank you!

**Comment by tmszi** [comment]
I tested [WinGRASS-8.0.0-1-Setup.exe](https://grass.osgeo.org/grass80/binary/mswindows/native/WinGRASS-8.0.0-1-Setup.exe) and bug still persist. When I want launch v.surf.rst module from CMD I get error message libgomp-1.dll, libgfortran-5.dll was not found.

**Comment by hellik** [comment]
Have you checked to install the MS runtimes libraries during installation?

**Comment by tmszi** [comment]
> Have you checked to install the MS runtimes libraries during installation?

Yes I did.

**Comment by hellik** [comment]
> > Have you checked to install the MS runtimes libraries during installation?
> 
> Yes I did.

any chance to test winGRASS 8 in OSGeo4W (https://trac.osgeo.org/osgeo4w/)?

**Comment by hellik** [comment]
> I tested [WinGRASS-8.0.0-1-Setup.exe](https://grass.osgeo.org/grass80/binary/mswindows/native/WinGRASS-8.0.0-1-Setup.exe) and bug still persist. When I want launch **v.surf.rst** module from CMD I get error message libgomp-1.dll, libgfortran-5.dll was not found.


confirmed in standalonrr and OSGeo4W:

**v.surf.rst** procudes these errors.

**Comment by hellik** [comment]
@landam any idea?

**Comment by hellik** [comment]
Missing libraries fixed  by #2204 

Closing issue for now. A separate issue will be opened regarding missing PROJ tools.
